samples/storage_csm_powerflex_v230.yaml

```diff
@@ -0,0 +1,133 @@
+apiVersion: storage.dell.com/v1alpha1
+kind: ContainerStorageModule
+metadata:
+  name: test-vxflexos
+  namespace: test-vxflexos
+spec:
+  driver:
+    configVersion: v2.3.0
+    replicas: 1
+    dnsPolicy: ClusterFirstWithHostNet
+    forceUpdate: false
+    forceRemoveDriver: true
+    fsGroupPolicy: File
```

Comment:
I believe fsGroup needs to be under a driverSpec section, like what was done w/ the 2.4 yaml:
 ```
  csiDriverSpec:
      # fsGroupPolicy: Defines if the underlying volume supports changing ownership and permission of the volume before being mounted.
      # Allowed values: ReadWriteOnceWithFSType, File , None
      # Default value: ReadWriteOnceWithFSType
      fSGroupPolicy: "File"
```

Reply:
changed it